src/Schema/ColumnDiff.php

```diff
@@ -31,6 +34,15 @@ public function __construct(
         array $changedProperties = [],
         ?Column $fromColumn = null
     ) {
+        if (func_num_args() < 4) {
```

Comment:
:thinking: so, passing `null` is not deprecated? That seems to contradict #4786 . Should this test be changed into a null check?

Reply:
Good point.